google/cloud/storage/examples/storage_notification_samples.cc

```diff
@@ -176,7 +182,7 @@ int main(int argc, char* argv[]) try {
 
   // Build the list of commands and the usage string from that list.
   using CommandType =
-      std::function<void(google::cloud::storage::Client, int&, char* [])>;
+      std::function<void(google::cloud::storage::Client, int&, char*[])>;
```

Comment:
In https://github.com/googleapis/google-cloud-cpp/blob/695675ce6c0ee169786e26388f4852032ba23235/google/cloud/storage/examples/storage_notification_samples.cc#L164

Question, Is `StatusOr<void>` not capable of `if (!var)`?